documentation/operators/src/nodes/maintenance.md

```diff
@@ -0,0 +1,699 @@
+```
+# for nym-mixnode, nym-gateway and nym-network requester:
+cat /proc/$(pidof <NODE>)/limits | grep "Max open files"` to make sure the limit has changed to 65535.
+
+# for validator
+cat /proc/$(pidof nym-validator)/limits | grep "Max open files"
+```
+Make sure the limit has changed to 65535.
+
+#### Set the ulimit on `non-systemd` based distributions
+
+In case you chose tmux option for mixnode automatization, see your `ulimit` list by running:
+
+```
+ulimit -a
+
+# watch for the output line -n
+-n: file descriptors          1024      
+```
+
+You can change it either by running a command:
+
+```
+ulimit -u -n 4096
+```
+
+or editing `etc/security/conf` and add the following lines:
+
+```
+# Example hard limit for max opened files
+username        hard nofile 4096
+
+# Example soft limit for max opened files
+username        soft nofile 4096
+```
+
+Then reboot your server and restart your mixnode.
+
+## Virtual IPs and hosting via Google & AWS
+For true internet decentralization we encourage operators to use diverse VPS providers instead of the largest companies offering such services. If for some reasons you have already running AWS or Google and want to setup a `<NODE>` there, please read the following.
+
+On some services (AWS, Google, etc) the machine's available bind address is not the same as the public IP address. In this case, bind `--host` to the local machine address returned by `$(curl ifconfig.me)`, but also specify `--announce-host` with the public IP. Please make sure that you pass the correct, routable `--announce-host`.
+
+For example, on a Google machine, you may see the following output from the `ifconfig` command:
+
+```
+ens4: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1460
+        inet 10.126.5.7  netmask 255.255.255.255  broadcast 0.0.0.0
+        ...
+```
+
+The `ens4` interface has the IP `10.126.5.7`. But this isn't the public IP of the machine, it's the IP of the machine on Google's internal network. Google uses virtual routing, so the public IP of this machine is something else, maybe `36.68.243.18`.
+
+`./nym-mixnode init --host 10.126.5.7`, initalises the mix node, but no packets will be routed because `10.126.5.7` is not on the public internet.
+
+Trying `nym-mixnode init --host 36.68.243.18`, you'll get back a startup error saying `AddrNotAvailable`. This is because the mix node doesn't know how to bind to a host that's not in the output of `ifconfig`.
+
+The right thing to do in this situation is to init with a command:
+```
+./nym-mixnode init --host 10.126.5.7 --announce-host 36.68.243.18
+```
+
+This will bind the mix node to the available host `10.126.5.7`, but announce the mix node's public IP to the directory server as `36.68.243.18`. It's up to you as a node operator to ensure that your public and private IPs match up properly.
+
+To find the right IP configuration, contact your VPS provider for support.
+
+## Nym API (previously 'Validator API') endpoints
```

Comment:
I'm not sure this is strictly 'maintenance' imo - probably deserves its own page.